warehouse/events/tags.py

```diff
@@ -123,7 +123,6 @@ class Project(EventTagEnum):
         OwnersRequire2FAEnabled = "project:owners_require_2fa:enabled"
         ProjectCreate = "project:create"
         ReleaseAdd = "project:release:add"
-        ReleaseFileRemove = "project:release:file:remove"
```

Comment:
Hmm, I didn't realize we already had this. Trying to decide if it makes more sense for this to stay the same (to maintain consistency) or change to what you have (to be more clear on what the event is generated from).